src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumChest.java

```diff
@@ -134,6 +135,25 @@ protected void addDisplayInformation(List<ITextComponent> textList) {
     public void addInformation(ItemStack stack, @Nullable World player, List<String> tooltip, boolean advanced) {
         super.addInformation(stack, player, tooltip, advanced);
         tooltip.add(I18n.format("gregtech.machine.quantum_chest.capacity", maxStoredItems));
+
+        NBTTagCompound compound = stack.getTagCompound();
+        if (compound != null) {
+            String translationKey = null;
+            long count = 0;
+            if (compound.hasKey("ItemStack")) {
+                translationKey = new ItemStack(compound.getCompoundTag("ItemStack")).getUnlocalizedName() + ".name";
```

Comment:
There is problem with showing names for some of our items as on picture below:

![2021-03-31_17 01 42](https://user-images.githubusercontent.com/3093101/113167589-28826e00-9244-11eb-97de-1525289dfe0b.png)
![2021-03-31_17 01 34](https://user-images.githubusercontent.com/3093101/113167593-29b39b00-9244-11eb-9675-7ac769047cae.png)